src/python/WMCore/ReqMgr/DataStructs/Request.py

```diff
@@ -70,8 +70,12 @@ def generateRequestName(request):
     current_time = time.strftime('%y%m%d_%H%M%S', time.localtime(time.time()))
     seconds = int(10000 * (time.time() % 1.0))
     request_string = request.get("RequestString", "")
+
     if request_string != "":
         request["RequestName"] = "%s_%s" % (request["Requestor"], request_string)
```

Comment:
I the example case, Requestor is pdmvserv@vocms081.cern.ch,  so pdmvserv is enough information that it is from host cert, right? I can change that.
 
Changing final case would be the same result unless there are other string includes '@' (Which shouldn't be the case anyway) Or we can created the lexicon for RequestString and prevent getting '@'. I guess you prefer second option